docs/docker.md

```diff
@@ -67,40 +71,80 @@ There are three kinds of guidelines included:
-touch with the FIWARE team at
-[fiware-help@lists.fiware.org](mailto:fiware-help@lists.fiware.org).
-
--   Dockerfiles and Dockerhub repositories **MUST** be linked from the FIWARE
-    Catalogue according to the Guidelines defined by the
-    [FIWARE Catalogue](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Working_with_the_FIWARE_catalogue#Creating_instances).
-
 -   The GitHub repository `README.md` **MUST** have a Docker reference - this is
     a link on the mandatory Docker Pulls `README.md` badge
 
 -   The **ReadtheDocs** GE Installation Documentation **MUST** include
     references to the Docker Hub image, and how to configure it.
 
+## Publication Requirements
+
+-   The GE owner will be responsible for Docker publication and maintenance
+    operations. Docker builds **SHOULD** be automated. GE Owner's build should
+    be triggered by git push, FIWARE's one will be triggered by the release of
+    new FIWARE-tagged versions.
+
 -   Docker containers **MUST** be tested before being published to the
     community. Error in Docker materials of a GE will be considered as critical
     and **MUST** be fixed immediately.
 
+-   Maintenance and release of interim point releases **MUST** be done using the
+    GE Owner own contributor account.
+
+-   The name of the `r/fiware/image` **MUST** match the standardized name of the
+    GitHub sub-module within the catalogue. It is not necessarily linked to the
+    name of the GitHub repository used by the team within GitHub. (e.g.
+    `apinf/platform` won’t create a FIWARE Docker image called `platform`.)
+
+-   A web-hook **MUST** be added so that the FIWARE Infrastructure is informed
+    whenever a release version is tagged. FIWARE will build tagged releases
+    only - i.e. `r/fiware/component/latest` will be the **latest stable
+    release**.
+
+-   Builds of components which do not successfully integrate with
+    `r/fiware/orion/latest` will not be added to the FIWARE account. When change
+    have been made, a [semVer](https://semver.org/) release on the source code
+    will be required in order to complete the release. This will kick off the
+    FIWARE build. Obviously it is essential that such a build passes the
+    mandatory integration tests.
+
+-   Dockerfiles and Dockerhub repositories **MUST** be linked from the FIWARE
+    Catalogue according to the Guidelines defined by the
+    [FIWARE Catalogue](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Working_with_the_FIWARE_catalogue#Creating_instances).
+
+## Recommended best practices
+
 -   Dockerfiles **SHOULD** follow best practices as described
     [here](https://docs.docker.com/articles/dockerfile_best-practices/)
 
--   Dockerfiles **SHOULD** be based on the latest LTS release of the base
-    image - e.g. `ubuntu:18.04`, `node:carbon` etc.
+-   Dockerfiles **SHOULD** be based on the latest LTS (Long Term Support)
```

Comment:
> Dockerfiles **SHOULD** be based on the latest LTS (Long Term Support) of the operating system. Where an image is based on a framework, the earliest LTS still receiving maintenance patches should be used

This reflects current practice within the IoT Agents, since we recommend node:8 (carbon) not node:10 ... yet.